mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -124,6 +138,10 @@ CLASS net/minecraft/class_113 net/minecraft/world/biome/Biome
 			ARG 2 variationModifier
 		METHOD method_548 diminish ()Lnet/minecraft/class_113$class_114;
 	CLASS class_115 Temperature
+		FIELD field_434 OCEAN Lnet/minecraft/class_113$class_115;
+		FIELD field_435 COLD Lnet/minecraft/class_113$class_115;
+		FIELD field_436 MEDIUM Lnet/minecraft/class_113$class_115;
+		FIELD field_437 WARM Lnet/minecraft/class_113$class_115;
```

Comment:
these are enum values
they get automatically mapped by stitch
hence they're grey in enigma

Reply:
And your point is?_

Reply:
> And your point is?

just saying, they don't need to be mapped

and all comments aren't for requesting changes. they're just comments